packages/documentation-v7/src/stories/components/collapsible/collapsible.docs.mdx

```diff
@@ -0,0 +1,63 @@
+import { Meta, Canvas, Story, Source, ArgsTable } from '@storybook/blocks';
+
+import * as CollapsibleStories from './collapsible.stories';
+
+<Meta of={CollapsibleStories} />
+
+# Collapsible
+
+<div className="lead">Toggle the visibility of content across your project.</div>
+
+The `<post-collapsible>` component is used to show and hide content.
+Collapsing an element will animate the height from its current value to 0.
+
+<Canvas of={CollapsibleStories.Default} />
+<ArgsTable story="^" />
+
+## Examples
+
+The following examples show different use cases for the `<post-collapsible>` component.
+
+### Initially Collapsed
+
+To make the collapsible content hidden by default, just use the `collapsible="true"` property.
+
+<Canvas of={CollapsibleStories.InitiallyCollapsed} />
+
+### Heading Level
+
+Use the `heading-level` property to define the hierarchical level of the collapsible header within the structure of a document.
+
+<Canvas of={CollapsibleStories.HeadingLevel} />
+
+### Intricate Content
+
+The collapsible component uses [HTML slot elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot)
+to allow any HTML content to be used as part of its header and body.<br/>
+By default, all children are appended to the component's body.
+For a specific child to be displayed in the header, a `slot="header"` attribute must be added.
+
+<Canvas of={CollapsibleStories.IntricateContent} />
+
+### Custom Trigger
+
+The `<post-collapsible>` component exposes a `.toggle()` method that allows to trigger the collapse programmatically.
+This method is asynchronous and returns a promise that resolves with the current open state.
+It optionally takes a boolean parameter that forces open when `true` or close when `false`.
+
+<Source
+  code={`const collapsible = document.querySelector('#collapsibleId') as HTMLPostCollapsibleElement;
+collapsible.toggle(/* open: boolean */).then((/* isOpen: boolean */) => {});`}
+  language="typescript"
+  dark={true}
+/>

```

Comment:
This seems to be the only way to use the dark theme at the moment. On dark theme, it is staying dark (like with the v6) version.

Reply:
It is good to know it's possible! Can you extract that in another PR?
For now we use light theme for all code blocks in v7 so its better to migrate all at once.

Reply:
Ok. I've created an issue: https://github.com/swisspost/design-system/issues/1677